# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/chkrootkit/make_install.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-12-26 18:34:10.000000000 -0300
+++ ./Makefile	2003-12-28 15:25:55.000000000 -0300
@@ -7,6 +7,8 @@
 CFLAGS	 = -DHAVE_LASTLOG_H
 STATIC   = -static
 
+SBIN     = /usr/sbin
+
 ###
 ### Solaris 2.x
 ###
@@ -63,3 +65,6 @@
 
 clean:
 	rm -f ${OBJS} core chklastlog chkwtmp ifpromisc chkproc chkdirs check_wtmpx strings-static
+
+install: sense
+	cp -vf chkrootkit chklastlog chkwtmp ifpromisc chkproc chkdirs check_wtmpx strings-static $(SBIN)
